src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -958,19 +958,6 @@ public function getModel()
     }
 
     /**
-     * Get a fresh instance of a model instance being queried.
-     *
-     * @param  array  $attributes
-     * @return \Illuminate\Database\Eloquent\Model
-     */
-    public function newModel(array $attributes = [])
```

Comment:
Hmmm. Let's let Taylor decide. I don't think merging this would be much of an issue tbh.